### ISSUE TITLE
Cannot find module 'react'

### ISSUE THREAD

**Issue by javiermarinros** [jackocnr/intl-tel-input#1556]
I am getting this error using this library (v21.0.3) from an Angular project:

```
✘ [ERROR] TS2307: Cannot find module 'react' or its corresponding type declarations. [plugin angular-compiler]

    node_modules/intl-tel-input/types.d.ts:449:33:
      449 │     import { ReactElement } from "react";
```

**Comment by jackocnr** [comment]
Thanks for reporting this. Can you try v21.0.5 and see if that fixes it?

If you're using Typescript and want the types, you may need to import them manually:

`import "intl-tel-input/build/js/types.d.ts";`

**Comment by javiermarinros** [comment]
Fixed with v21.0.5. Thank you!
